www-apps/phpsysinfo/phpsysinfo-3.3.0.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils webapp
+
+DESCRIPTION="A customizable PHP script that displays information about your system nicely"
+HOMEPAGE="https://rk4an.github.com/phpsysinfo/"
+SRC_URI="https://github.com/rk4an/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+KEYWORDS="~amd64 ~x86"
+RDEPEND="
+	dev-lang/php[simplexml,xml,xsl(+),xslt(+),unicode]
+	virtual/httpd-php
+"
+
+need_httpd_cgi
+
+src_install() {
+	webapp_src_preinst
+
+	dodoc CHANGELOG.md README*
+	rm CHANGELOG.md COPYING README* .gitignore .travis.yml || die
+
+	insinto "${MY_HTDOCSDIR}"
+	doins -r .
+	newins phpsysinfo.ini{.new,}
```

Comment:
Does this mean you're installing both `phpsysinfo.ini` and `phpsysinfo.ini.new`?

Reply:
No? It installs phpsysinfo.ini.new as phpsysinfo.ini. If you wan't I can make it better readable?

Reply:
I meant: doesn't `doins -r` above install it as `.new`?

Reply:
Yes, this would install ```phpsysinfo.ini.new```. Well, upstream ships only ```phpsysinfo.ini.new```, but to use phpsysinfo, you need to have ```phpsysinfo.ini```. So also installing ```phpsysinfo.ini.new``` as ```phpsysinfo.ini```, which a user can configure, but always can have a look into defaults upstream? Not a good idea?

Reply:
I'm asking if there's a sense in having both installed rather than one of them only.

Reply:
Depends, I guess. I think it could be. But maybe, this confuses some users. Changed, to move .new to .ini.